packages/next/client/url.js

```diff
@@ -0,0 +1,70 @@
+import { encode, decode } from 'querystring'
```

Comment:
This ends up being handled by the `querystring` alias for sure, right?

Reply:
Yes. I'm aliasing `querystring` only in the client side webpack config